### PR TITLE
feat: support relative app icons in appDetails, closes #348

### LINKED ISSUE #348: Connect to support relative icon URL
Can Connect allow a relative URL for `appDetails.icon`?

That way, when somebody forks an app template like [REBL One](https://github.com/REBL-Stack/starter-app) that uses a relative icon URL, the app icon is served from their clone as they should expect, rather from some hardcoded server.

### PR DESCRIPTION
**Pull request by hstove**

### CODE REVIEW

**Review by kyranjamie — APPROVED** [review]

**Comment by markmhendrickson** [comment]
Hank still needs to get deployment working then Tim can QA this.

File: packages/test-app/components/app.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { ThemeProvider, Box, theme, Text, Flex, CSSReset, Button, Stack } from '
 import { Connect, AuthOptions, useConnect } from '@blockstack/connect';
 import { UserSession, AppConfig } from 'blockstack';
 
-const icon = `${document.location.origin}/assets/messenger-app-icon.png`;
+const icon = `/assets/messenger-app-icon.png`;
```
Comment (aulneau) [inline packages/test-app/components/app.tsx]: Should not be `, but single quotes

**Review by aulneau — APPROVED** [review]

**Comment by markmhendrickson** [comment]
@timstackblock it appears deployment is now working with https://deploy-preview-459--authenticator-demo.netlify.app/

**Comment by timstackblock** [comment]
@hstove I am able to click this link and login but the icon is not displaying.

**Comment by timstackblock** [comment]
I think this error only appears when you use the chrome extension.

**Comment by hstove** [comment]
Yes - if you use the latest extension build for this PR, it should work. But it doesn't work with the current production extension.

**Pull request merged**